api/src/main/java/com/github/retrooper/packetevents/wrapper/play/client/WrapperPlayClientSelectTrade.java

```diff
@@ -36,18 +36,17 @@ public WrapperPlayClientSelectTrade(int slot) {
 
     @Override
     public void read() {
-        //1.13+ packet
```

Comment:
why was this comment removed?

Reply:
Why should it be there? It's neither JavaDocs nor any useful information for one single class, though he can check it in our ClientboundPacketType classes.